Do all supported platforms use python3 as the "default" python? I'm getting:
```
Traceback (most recent call last):
  File "/var/lib/buildbot/slave/sage_git/build/build/bin/sage-package", line 40, in <module>
    from sage_bootstrap.cmdline import run
  File "/var/lib/buildbot/slave/sage_git/build/build/bin/../sage_bootstrap/cmdline.py", line 31, in <module>
    from sage_bootstrap.app import Application
  File "/var/lib/buildbot/slave/sage_git/build/build/bin/../sage_bootstrap/app.py", line 80
    def properties(self, *package_classes, props=['path', 'version_with_patchlevel', 'type', 'source', 'trees'],
                                               ^
SyntaxError: invalid syntax
************************************************************************
Error downloading tarball of ecl
************************************************************************
```

--- a/build/sage_bootstrap/app.py
+++ b/build/sage_bootstrap/app.py
@@ -77,8 +77,7 @@ class Application(object):
         for pkg_name in pc.names:
             print(pkg_name)
 
-    def properties(self, *package_classes, props=['path', 'version_with_patchlevel', 'type', 'source', 'trees'],
-                   format='plain'):
+    def properties(self, *package_classes, **kwds):
         """
         Show the properties of given packages
 
@@ -89,6 +88,8 @@ class Application(object):
         source_maxima='normal'
         trees_maxima='SAGE_LOCAL'
         """
+        props = kwds.pop('props', ['path', 'version_with_patchlevel', 'type', 'source', 'trees'])
+        format = kwds.pop('format', 'plain')
         log.debug('Looking up properties')
         pc = PackageClass(*package_classes)
         for package_name in pc.names:

